python/flare/shellcode_hash_search.py

```diff
@@ -467,17 +468,19 @@ def promptForHashTypes(self):
         '''
         # Only run if QT not available, so not bothering with ida7 check
         hashTypes = self.dbstore.getAllHashTypes()
-        if len(self.params.hashTypes) == 0:
-            raise RuntimeError('No hashes selected')
-        # we used to prompt y/n for each one. too obnoxious, just force all hashes
+        
+		#if len(self.params.hashTypes) == 0:
+        #    raise RuntimeError('No hashes selected')
+        
+		# we used to prompt y/n for each one. too obnoxious, just force all hashes
```

Comment:
Go ahead & delete the commented out code & update the PR please, then I'll approve the merge.